lib/matplotlib/path.py

```diff
@@ -1,23 +1,19 @@
-"""
-A module for dealing with the polylines used throughout matplotlib.
+r"""
+A module for dealing with the polylines used throughout Matplotlib.
 
-The primary class for polyline handling in matplotlib is :class:`Path`.
-Almost all vector drawing makes use of Paths somewhere in the drawing
-pipeline.
+The primary class for polyline handling in matplotlib is `Path`.  Almost all
+vector drawing makes use of `Path`\s somewhere in the drawing pipeline.
```

Comment:
For my information, why is the backslash needed here?  And should "matplotlib" be capitalized in the line above?

Reply:
Fixed capitalization.  Backslash is needed because a word-internal backquote is not recognized per item 7 of http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#inline-markup-recognition-rules.